pkg/cmd/oauth-apiserver/openapiconfig/openapiconfig.go

```diff
@@ -115,3 +116,101 @@ func DefaultOpenAPIConfig() *openapicommon.Config {
 		SecurityDefinitions: &securityDefinitions,
 	}
 }
+
+func DefaultOpenAPIV3Config() *openapicommon.OpenAPIV3Config {
+	defNamer := apiserverendpointsopenapi.NewDefinitionNamer(legacyscheme.Scheme, extensionsapiserver.Scheme, aggregatorscheme.Scheme)
+	return &openapicommon.OpenAPIV3Config{
+		Info: &spec.Info{
```

Comment:
It looks like `spec.Info` could be shared between `DefaultOpenAPIV3Config` and `DefaultOpenAPIConfig`, could you refactor ?